windows_set_certificate_friendlyname/PF_StartupScript.ps1

```diff
@@ -0,0 +1,40 @@
+<#
+.SYNOPSIS
+  <Overview of script>
+.DESCRIPTION
+  <Brief description of script>
+.NOTES
+  Version:        1.0
+  Author:         <Name>
+  Creation Date:  <Date>
+  Purpose/Change: Initial script development
+#>
+
+# Set Error Action to Stop
+$ErrorActionPreference = "Stop"
+
+# Grab the script path 
+$scriptPath = split-path -parent $MyInvocation.MyCommand.Definition
+echo "the current script path is $scriptPath"
+
+# these are some of the env variables that are available to you
+echo "PlayFab Title ID is $env:PF_TITLE_ID" # e.g. 59F84
+echo "PlayFab Build ID is $env:PF_BUILD_ID" # Guid, e.g. 09d91059-22d3-4d0a-8c99-f34f80525aae
+echo "PlayFab Virtual Machine ID is $env:PF_VM_ID" # e.g. vmss:SouthCentralUs:2458795A9259968E_12fe54be-fae1-41aa-83d9-09b809d5ef01:09d91059-22d3-4d0a-8c99-f34f80525aae
+echo "Region where the VM is deployed is $env:PF_REGION" # e.g. SouthCentralUs
+echo "Shared content folder is $env:PF_SHARED_CONTENT_FOLDER_VM" # e.g. D:\sharedcontentfolder (All servers running on this VM have access to this folder through the PF_SHARED_CONTENT_FOLDER env variable.)
+
+# ACTION: Specify the correct subject name for your certificate
+$subjectName = "DC=..., O=..., OU=..., CN=..."
+
+# Find the certificate by subject name in the machine store
+$cert = Get-ChildItem -Path Cert:\LocalMachine\My | Where-Object { $_.SubjectName.Name -eq $subjectName }
+
+if ($cert) {
+    # ACTION: specify a proper friendly name for your certificate
+    $cert.FriendlyName = "My cert friendly name"
+    $cert | Out-Null  # Suppress output
+    Write-Host "Friendly name set to 'My cert friendly name' for the certificate with subject name: $subjectName"
```

Comment:
can you use the $cert.FriendlyName variable here?

Reply:
tried, but the output is getting weird (all the properties of the cert are sent to output), not sure how to suppress it. Happy to hear any ideas